change/@ni-nimble-angular-1d94428d-84dd-4747-bab0-83d68689a39f.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
I probably missed the conversation where this was discussed. Could I get some more context/rationale?

At first glance the new name sounds weird to me. i.e. a group of radios rather than a group of buttons on a radio where only one button can be pressed at once.

Reply:
Main issue brought up was inconsistency should either be:

nimble-radio-button-group + nimble-radio-button
or
nimble-radio-group + nimble-radio

But the current name is inconsistent. Fast uses fast-radio and fast-radio-group for their fast component implementation. Carbon went with RadioButtonGroup and RadioButton.

Went with `radio-group` and `radio` as when brainstorming with @m-akinc and @atmgrifter00 we thought it would be uncontroversial but we can bring it up at team meeting if we should have team agreement or there are other concerns.

Reply:
From openui https://open-ui.org/components/radio-button.research
![image](https://user-images.githubusercontent.com/1588923/193363203-052ebbe7-620e-4145-bafe-3973ce6352cb.png)

(not to argue one way or the other, just showing more examples)

Reply:
Thanks. I think we should discuss as a group or at least let @nate-ni weigh in.

Reply:
I don't have a strong preference. 

I accept that even though it is inconsistent with things like checkboxes or switches, "radio button" is commonly used when we discuss them. However, it would be shorter and more consistent to remove the "button" part.

Ultimately, do what you think is right. I'm down either way.

Reply:
In the team meeting today we decided on "radio" and "radio group". 📻